### ISSUE TITLE
Can't download Boxarts

### ISSUE THREAD

**Issue by Ritvikisalive-coding** [KirovAir/TwilightBoxart#20]
i used it a couple of weeks ago and it worked but i tried today because i added more games to my Twilight Menu ++ but all i get is " An error occurred while sending the request. The remote name could not be resolved: 'boxart.kirovair.com' "

**Comment by getBoolean** [comment]
Duplicate of https://github.com/KirovAir/TwilightBoxart/issues/17

**Comment by LeyendaV** [comment]
Still no fix for this.

**Comment by varchild** [comment]
Bring your website back online or give us the ability to input different sites to pull box art from.

**Comment by Masamune3210** [comment]
Who are you to demand anything from a tool provided for free?

**Comment by TC-Wheel** [comment]
This website has all the box arts: https://thumbnails.libretro.com/Nintendo%20-%20Nintendo%20DS/Named_Boxarts/
This isn't a fix or anything- just if the dev or someone else wants to rewrite this to pull boxarts.

**Comment by robcmo** [comment]
Yes, the domain boxart.kirovair.com is no longer registered to the author. Unfortunately, the program was coded to use his website instead of GameTDB.

**Comment by eku** [comment]
> Unfortunately, the program was coded to use his website instead of GameTDB.

You are free to fork the repo and change the download function.

**Comment by getBoolean** [comment]
@robcmo as others have mentioned before, downgrade to version 0.6 which did not use his server

**Comment by robcmo** [comment]
Thank you @getBoolean I'll consider trying that.
I already extracting the files from the US archive [S Covers (png)](https://www.gametdb.com/DS/Downloads) which is the correct size and has pretty much all I want anyway.

**Comment by taylorprod** [comment]
version 0.6 works! 
It has a db of artwork included so it doesnt rely on making a request to an external service.

**Comment by KirovAir** [comment]
That was the server dying, sorry! It's back as of TwilightBoxart 2.0, and your existing exe should work again as-is. The new version identifies games a lot better though, worth the upgrade: https://github.com/KirovAir/TwilightBoxart/releases/tag/2.0.0 :)
